syntax = "proto3";
package proto;
import "Reserva.proto";
option java_package = "sd.ufc.reserva.model";
option java_outer_classname = "ObjectListaReserva";

message ListaReserva{
    repeated Reserva reservas=1; 
}